server/command.go

```diff
@@ -149,6 +164,8 @@ func (p *Plugin) executeStartMeetingCommand(c *plugin.Context, args *model.Comma
 		}
 	}
 
+	p.trackEvent(args, "meeting-link", p.configuration.JitsiURL)
```

Comment:
I don't think the full URL should be tracked as it might contain confidential information. Only _if_  the default value is used should be tracked.